sys/src/cmd/6l/BUILD

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+mkdir -p ../../../../linux_amd64/bin
```

Comment:
I'd greatly prefer that this just creates a binary called `6l.linux_elf64` or something and then we can copy it wherever needed.